data/graphics.js

```diff
@@ -10009,11 +10058,60 @@ var BattleMoveAnims = {
 			}, 'decel', 'explode');
 		}
 	},
+	hydroshot: {
```

Comment:
The idea was to use Hydro Pump for this animation, but when I was trying to put it in, errors would show up, dunno why. that's why I just c/p'd it. If I actually get it to work, I'll update it